### PR TITLE
Fixes to 1.5.0-beta

### LINKED ISSUE #83: new issues and feature request
**Describe the bug**
bot was setup for a 1v1 but after readying up the bot froze at map veto. 

need an option to empty the queue and delete the vc for the match

### PR DESCRIPTION
**Pull request by yannickgloster**
- Player veto count now is based off of match size
- Restarts the queue when the size of the queue is changed
- Add command to forcefully restart the queue
- Added the option to set queue captains
   - You can set as many as you'd like, just tag them, once they have been a captain, they will be removed from the queue captains list.
- Added a command to restart the whole bot. Only use this if it breaks, I would not recommend using it though and I would say restart the bot with the code manually bc this will create a process that is hard to find, but the option is there if you need it.
- Added a command to kick everyone from the queue
- Added a check for the end of round 16 to speed up the ending of games

Closes #83

### CODE REVIEW

File: utils/server.py [hunk 1]
```diff
@@ -116,8 +117,10 @@ async def _handler(self, request: web.Request) -> web.Response:
                                               inline=False)
                     score_embed.set_footer(text="🟢 Live")
                     await server.score_message.edit(embed=score_embed)
+                    if get5_event["params"]["team1_score"] == 16 or get5_event["params"]["team2_score"]:
```
Comment (lexesj) [inline utils/server.py]:
```suggestion
                    if get5_event["params"]["team1_score"] == 16 or get5_event["params"]["team2_score"] == 16:
```
Typo

File: cogs/csgo.py [hunk 2]
```diff
@@ -82,6 +82,14 @@ async def pug(self, ctx: commands.Context, *args):
                 else:
                     raise commands.CommandError(message=f'Invalid Argument: `{arg}`')
 
+        if not self.pug.enabled:
+            if len(self.bot.queue_captains) > 0:
+                team1_captain_arg = self.bot.queue_captains[0]
+                self.bot.queue_captains.remove(team1_captain_arg)
```
Comment (lexesj) [inline cogs/csgo.py]:
```suggestion
                team1_captain_arg = self.bot.queue_captains.pop(0)
```
Can be changed to this

File: cogs/csgo.py [hunk 3]
```diff
@@ -139,6 +147,16 @@ async def pug(self, ctx: commands.Context, *args):
...
 
+            player_veto = []
+            if self.bot.match_size == 2:
+                player_veto = [1, 1]
+            for i in range(self.bot.match_size - 2):
+                if i == 0 or i == self.bot.match_size - 3:
+                    player_veto.append(1)
+                elif i % 2 == 0:
+                    player_veto.append(2)
+            player_veto = player_veto + [1, 1]
```
Comment (lexesj) [inline cogs/csgo.py:156]: If the match size is 2, the player_veto would be `[1,1,1,1]`, not sure if this is is intended or not

File: cogs/csgo.py (hunk 2 above)
Comment (yannickgloster) [reply cogs/csgo.py]: fixed in latest

File: cogs/csgo.py (hunk 3 above)
Comment (yannickgloster) [reply cogs/csgo.py:156]: This is intended. I add [1, 1] to end of every veto for coaches. It'll never be used though in the queue and will only be used if you run .pug with more people than the match size

**Comment by yannickgloster** [comment]
Waiting for @lexesjan

**Review by lexesj — APPROVED** [review]

**Pull request merged**